sktime/dists_kernels/dtw/_dtw_python.py

```diff
@@ -6,7 +6,7 @@
 
 import numpy as np
 
-from sktime.distances import pairwise_distance
+from sktime.distances._distance import pairwise_distance
```

Comment:
Why is this required? It's being reexported in the `__init__` moduke, is this insufficient?

https://github.com/sktime/sktime/blob/d4a0dc3f9a622b673bc385a83a8271f5fe99b81c/sktime/distances/__init__.py#L50